extras/indexing/src/main/java/mvm/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -67,6 +67,10 @@
 
 /**
  * A set of configuration utils to read a Hadoop {@link Configuration} object and create Cloudbase/Accumulo objects.
+ * Soon will deprecated this class.  Use installer for the set methods, use RyaDetails for the get methods. 
```

Comment:
I don't think he can link the Install class right now because that is in RYA-82.